CRM/Core/PseudoConstant.php

```diff
@@ -1650,6 +1650,10 @@ public static function countryIDForStateID($stateID) {
    *   array reference of all greetings.
    */
   public static function greeting($filter, $columnName = 'label') {
+    if (!isset(Civi::$statics[__CLASS__]['greeting'])) {
```

Comment:
All the changes in CRM/Core/PseudoConstant.php are already merged, apparently in PR #11313. I think that explains the "merge conflict" error in the Jenkins checks.

Reply:
This is the only thing that stands out to me, @eileenmcnaughton . Other changes look sensible, I can't think of any way to improve them. 

But oddly, I'm not seeing the E_NOTICE you mention, using code like you provided, on master:

```
  public function testNotice() {
    // This will cause a failing test because of the NOTICE-level error:
    // trigger_error('Notice.', E_USER_NOTICE);
    // But with that line commented out, the test passes just fine.

    $contact_id = $this->individualCreate();    
    $contact = $this->callAPISuccess('Contact', 'create', array(
     'contact_type' => 'Individual',
     'id' => $contact_id,
     'postal_greeting_id' => 2,
    ));
  }
```